docs/tutorials/installation.md

```
@@ -0,0 +1,64 @@
+title: "Step-by-Step Installation Tutorial"
+description: "Beginner-friendly tutorial to install a single-node Elastic Stack with unprivileged containers."
+nav_order: 30
+---
+
+# Step-by-Step Installation Tutorial
+
+This step-by-step tutorial teaches you how to deploy a single-node instance of Elasticsearch and Kibana utilizing hardened Wolfi images inside an isolated, rootless Podman network.
+
+---
+
+## 🎓 Learning Objectives
```

markdownlint-cli2 (0.23.2)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)